src/main/groovy/com/blackducksoftware/integration/hub/detect/hub/BdioUploader.groovy

```diff
@@ -47,14 +47,14 @@ class BdioUploader {
     void uploadBdioFiles(HubServerConfig hubServerConfig, BomImportRequestService bomImportRequestService, PhoneHomeDataService phoneHomeDataService, List<File> createdBdioFiles) {
         createdBdioFiles.each { file ->
             logger.info("uploading ${file.name} to ${detectConfiguration.getHubUrl()}")
-            bomImportRequestService.importBomFile(file, BuildToolConstants.BDIO_FILE_MEDIA_TYPE)
+            bomImportRequestService.importBomFile(file, 'application/ld+json')
```

Comment:
hub-common should be encapsulating this - I'll add a new method we can use

Reply:
Let me know where you add it to so that I can find it quickly and fix this.